backend/src/packages/chaiNNer_standard/utility/color/separate_color.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import annotations
+
+from enum import Enum
+
+from nodes.impl.color.color import Color
+from nodes.properties.inputs import ColorInput, EnumInput
+from nodes.properties.outputs import AnyOutput
+from nodes.utils.utils import round_half_up
+
+from .. import color_group
+
+
+def _to_num(n: float) -> int:
+    return round_half_up(max(min(n * 255, 255), 0))
+
+
+class SeparateColorMode(Enum):
+    UINT8 = 0
+    PERCENT = 1
+    COLOR = 2
+
+
+LABELS = {
+    SeparateColorMode.UINT8: "Uint8",
+    SeparateColorMode.PERCENT: "Percent",
+    SeparateColorMode.COLOR: "Color",
+}
```

Comment:
Uint8 and Percent aren't good names. People that aren't programmers don't know what a uint8 is, and a percentage is typically a number between 0 and 100.

I would suggest labelling uint8 as "8 Bit". Not sure what to do about percentage tbh.

Reply:
I could make percent an int between 0 and 100 instead?

Reply:
Idk tbh. Bot 0-1 and 0-100 would be useful.

Reply:
I think 0-100 would be better. It can always just be divided by 100 after all